pydsa/tests/test_queue.py

```diff
@@ -0,0 +1,16 @@
+from pydsa.queue import Queue
+
+
+def test_queue():
+    def __init__self(self, size):
+        self.queue = Queue()
+        queue.size = 5
+    def test_enqueue(self):
+        self.queue.enqueue(3)
+        self.queue.enqueue(7)
+        self.queue.enqueue(13)
+        self.queue.enqueue(9)
+        self.assertEqual(size() == 3)
```

Comment:
Please see the documentation on how the tests are written in PyDSA.
https://github.com/pydsa/pydsa/wiki/Writing-and-Running-Tests-in-PyDSA